tasks/configure.js

```diff
@@ -1,33 +1,37 @@
 'use strict';
 
-let packageJson = require(process.cwd() + '/package.json');
-let findService = require('../lib/find-service');
-let herokuAuthToken = require('../lib/heroku-auth-token');
-let normalizeName = require('../lib/normalize-name');
-let vault = require('../lib/vault');
-let fetchres = require('fetchres');
+const packageJson = require(process.cwd() + '/package.json');
+const findService = require('../lib/find-service');
+const herokuAuthToken = require('../lib/heroku-auth-token');
+const normalizeName = require('../lib/normalize-name');
+const vault = require('../lib/vault');
+const pipelines = require('../lib/pipelines');
+const HerokuConfigVars = require('../lib/heroku-config-vars');
 
 const FORBIDDEN_ATTACHMENT_VARIABLES = [
 	'DATABASE_URL'
 ];
 
 const DEFAULT_REGISTRY_URI = 'https://next-registry.ft.com/v2/';
 
-const getServiceData = (source, registry) => fetch(registry)
-	.then(response => response.json())
-	.then(json => {
-		const serviceData = findService(json, normalizeName(source));
-		if (!serviceData) {
-			throw new Error('Could not find a service in the registry, with `name` or `systemCode`, matching ' + source + '. Please check the service registry.');
-			return false;
-		}
-		else {
-			return serviceData;
-		}
-	});
+const getServiceData = (source, registry) => {
```

Comment:
this could be `(source, registry) => fetch(registry)...` as it's just returning straight away.